utils/src/treev2/quickPickWizard/ContextValueQuickPickStep.ts

```diff
@@ -20,14 +22,15 @@ export class ContextValueQuickPickStep<TNode extends types.ContextValueFilterabl
             (Array.isArray(excludeOption) ? excludeOption : [excludeOption]) :
             [];
 
-        const nodeContextValues: string[] = node.quickPickOptions.contextValues;
+        const nodeContextValues: string[] = parseContextValue(node.contextValue);
 
         return includeArray.some(i => this.matchesSingleFilter(i, nodeContextValues)) &&
             !excludeArray.some(e => this.matchesSingleFilter(e, nodeContextValues));
     }
 
-    protected override isIndirectPick(node: TNode): boolean {
-        return node.quickPickOptions.isLeaf === false;
+    protected override isIndirectPick(node: TreeItem): boolean {
+        // TreeItemCollapsibleState.None is falsy

```

Comment:
Maybe elaborate here why any falsy value means not an indirect pick?
```suggestion
        // `TreeItemCollapsibleState.None` and `undefined` are both falsy, and indicate that a `TreeItem` cannot have children--and therefore, cannot be an indirect pick
```